### PR TITLE
Caasp merge

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

**Comment by jreidinger** [comment]
@lslezak what repo you use for SP2 docker? it probably missing newer CFA

**Comment by teclator** [comment]
The repo is the OBS one, so @lslezak already fixed an issue syncing them, and after some rebuild and gitignore update Travis is not complaining anymore.

File: package/yast2-packager.changes [hunk 1]
```diff
@@ -22,6 +22,19 @@ Tue Mar  7 09:37:15 CET 2017 - schubi@suse.de
 - 3.1.122
 
 -------------------------------------------------------------------
+Fri Mar 24 13:53:40 UTC 2017 - mvidner@suse.com
```
Comment (lslezak) [inline package/yast2-packager.changes]: Is it OK to change the history for a maintained package?

Comment (teclator) [inline package/yast2-packager.changes]: It is the CASP history, so we have the SP2 history and the CASP history, which one should be the one selected if for example the next maintenance update is for CASP?

Comment (lslezak) [inline package/yast2-packager.changes]: I see, good point. So I'm fine with the change.

**Pull request merged**